using System;









// public class ExceptionHandling
// {
//     public static void Main(string[] args)
//     {
//         try
//         {
//             int a=100,b=4; 
//             int c=a+b;
//             Console.WriteLine("Asapu+...+");
//         }
//         catch(Exception e)
//         {
//             Console.WriteLine(e);
//         } 
//         finally
//         {
//             Console.WriteLine("Sri+...+");
//         }
//         Console.WriteLine("kumar+...");
//         Console.WriteLine("Manikanta+...");

//     }
// }











// public class AgeCalculatorException : Exception
// {
//     public AgeCalculatorException(string message) : base(message)
//     {
        
//     }
// }
// public class User
// {
//     static void validate(int age)
//     {
//         if(age<18)
//         {
//             throw new AgeCalculatorException("sorry,Age be greater than 18");
//         }
//     }
//     public static void Main(string[] args)
//     {
//         try
//         {
//          validate(12);
//         }
//         catch(AgeCalculatorException e)
//         {
//             Console.WriteLine(e);
//         }
//         Console.WriteLine("rest of the code");
//   }
// }









// public class ExceptionHandling
// {
//     public static void Main(string[] args)
//     {
//         try
//         {
//             int a=100,b=0; 
//             int c=a/b;
//         }
//         catch(Exception e)
//         {
//         Console.WriteLine(e);
//         }
//     }
// }










// public class ExceptionHandling
// {
//     public static void Main(string[] args)
//     {
//         try
//         {
//             int a=100,b=; 
//             int c=a+b;
//         }
//         catch(Exception e)
//         {
//             Console.WriteLine(e);
//         } 
//         Console.WriteLine("try and catch block program");
//     }
// }
